test/browser.js

```diff
@@ -73,22 +75,24 @@ test('throws TimeoutError even though it does not support AbortController', with
 		response.end();
 	});
 
-	server.get('/endless', () => {});
+	server.get('/slow', (request, response) => {
+		setTimeout(() => {
+			response.end('ok');
+		}, 1000);
+	});
 
 	await page.goto(server.url);
 	await page.addScriptTag({path: './test/helpers/disable-abort-controller.js'});
 	await page.addScriptTag({path: './umd.js'});
 
-	// TODO: make set a timeout for this evaluation so we don't have to wait 30s
 	const error = await page.evaluate(url => {
 		window.ky = window.ky.default;
 
-		const request = window.ky(`${url}/endless`, {timeout: 500}).text();
+		const request = window.ky(`${url}/slow`, {timeout: 500}).text();
 		return request.catch(error_ => error_.toString());
 	}, server.url);
 	t.is(error, 'TimeoutError: Request timed out');
 
-	// A note from @szmarczak: await server.close() hangs on my machine
```

Comment:
It was hanging for me, too. I think because the server's `response` stream was never being closed. That behavior doesn't seem necessary for this test, so I simply made the Ky timeout less than the server timeout and it works as expected.

Reply:
Thank you! :relaxed: